tests/sndrcv_failover.sh

```diff
@@ -9,7 +9,62 @@
 # This file is part of the rsyslog project, released  under GPLv3
 echo ===============================================================================
 echo \[sndrcv_failover.sh\]: testing failover capabilities for tcp sending
-. $srcdir/sndrcv_drvr_noexit.sh sndrcv_failover 50000
+
+# uncomment for debugging support:
+. $srcdir/diag.sh init
+# start up the instances
+#export RSYSLOG_DEBUG="debug nostdout noprintmutexaction"
+export RSYSLOG_DEBUGLOG="log"
+generate_conf
+export PORT_RCVR="$(get_free_port)"
+add_conf '
+$ModLoad ../plugins/imtcp/.libs/imtcp
+# then SENDER sends to this port (not tcpflood!)
+$InputTCPServerRun `echo $PORT_RCVR`
+
+$template outfmt,"%msg:F,58:2%\n"
+$template dynfile,`echo $RSYSLOG_OUT_LOG` # trick to use relative path names!
+:msg, contains, "msgnum:" ?dynfile;outfmt
+'
+startup
+. $srcdir/diag.sh wait-startup
+export RSYSLOG_DEBUGLOG="log2"
+#valgrind="valgrind"
+generate_conf 2
+export TCPFLOOD_PORT="$(get_free_port)" # TODO: move to diag.sh
+add_conf '
+$ModLoad ../plugins/imtcp/.libs/imtcp
+# this listener is for message generation by the test framework!
+$InputTCPServerRun `echo $TCPFLOOD_PORT`
+
+*.*	@@127.0.0.1:13516 # this must be DEAD
+$ActionExecOnlyWhenPreviousIsSuspended on
+&	@@127.0.0.1:`echo $PORT_RCVR`
```

Comment:
Backticks are only supported in new-style objects. However, you can work around this by using .... 1:'$PORT_RCVR'